.changeset/tricky-cups-camp.md

```diff
@@ -0,0 +1,7 @@
+---
+'@twilio-paste/input': patch
+'@twilio-paste/input-box': patch
+'@twilio-paste/core': patch
+---
+
+[Input, Input Box] Update styles to align with new Paste Twilio theme
```

Comment:
Nit (non-blocking): it doesn't look like theres any changes to the input package anymore, so you can probs just make this a patch for input-box

Reply:
oh good call, fixed it up!